TurkishFilter/sections/specific.txt

```diff
@@ -58,6 +58,7 @@ gazetebirlik.com###masthead-reklam
 gazetebirlik.com##.reklam-button
 acikradyo.com.tr##.reklam-item
 teknodestek.com.tr##div[style="float: none; margin:10px 0 10px 0; text-align:center;"]
+cimri.com##div[class] > a[href^="https://www.ciceksepeti.com/?"]
```

Comment:
why not just `a[href^="https://www.ciceksepeti.com/?"]`?

Reply:
They have already changed url.